qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -28,13 +28,14 @@ class SuperstaqBackend(qiskit.providers.BackendV1):
     """This class represents a Superstaq backend."""
 
     def __init__(self, provider: qss.SuperstaqProvider, target: str) -> None:
-        """Initializes a SuperstaqBackend.
+        """Initializes a `SuperstaqBackend`.
```

Comment:
Is this something we're meant to do? If so, let's add it to the dev guide. I myself have not been code-styling objects.

Reply:
We definitely use backticks for code snippets like `qiskit.QuantumCircuit` for example. But as the docstrings have been updated, the standard seems to be that `args` and `objects` should be code-styled. Alternatively, we could use `:class:` and `:attr:`, but I haven't seen any instance of that. 

Though maybe @vtomole @richrines1 can clarify what guideline should be added to the dev guide.